.github/workflows/ci.yml

```diff
@@ -9,4 +9,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - id: tests
-        run: ./scripts/tests.sh
\ No newline at end of file
+        run: |
+          sudo apt-get install libsndfile-dev ffmpeg
```

Comment:
why is this needed here and not in the `Dockerfile`?

Reply:
loading audio with librosa is performed during unit tests therefore installing libsndfile-dev and ffmpeg is needed.
However, been thinking that the executor itself doesn't need to load the images so it's not need so that the executor works.
What do you think ?

Reply:
good point